edu_robot/State state
edu_robot/Mode mode
string info_message
